package.json

```diff
@@ -20,7 +20,7 @@
   },
   "type": "module",
   "engines": {
-    "node": ">=16.0.0 <22.0.0"
+    "node": ">=18.20.0"
```

Comment:
Prior to Node 18.20 it did not support import attributes, which our code now uses.